scrnaseq.Rmd

```diff
@@ -277,11 +279,14 @@ knitr::kable(head(sc@meta.data, 5), align="l", caption="Meta-data, top 5 rows")
 
 ```
 
-```{r qc_plot}
+```{r qc_filter}
 # Filter cutoffs
 cells_filter = c(nFeature_RNA=200, nCount_RNA=NA, percent.mt=20)
 features_filter = c(min_counts=1, min_cells=1) # feature has to be found by at least one count in one cell 
+```
 
+We keep cells with more than `r cells_filter["nFeature_RNA"]` detected features, and cells with less than `r cells_filter["percent.mt"]`% of mitochondrial reads. We keep features detected with at least `r features_filter["min_counts"]` count(s) in at least `r features_filter["min_cells"]` cell(s). 
```

Comment:
Here we need a solution for multiple datasets as there may well be different filtering criteria. Current implementation is a list with dataset names and then the filter criteria. Also there may be min and max values.

Reply:
Yes. Can we leave it for this PR, since this belongs to the branch for multiple samples you are working on right now?

Reply:
But how?

Reply:
What 'but how'? How we want to mention the filters in the future once we support multiple samples with different filtering cutoffs?

Reply:
Yes, exactly. I would need an idea here.

Reply:
Can we solve this outside this PR, since it is related to your branch? 

We can create a string in a small chunk before this, for example, that contains filter parameters per sample. We can then print the string in the description of this chunk. If you show me again the exact format of the new filter parameters, I can come up with something for you.

Reply:
okay then let us do it like this.